packages/appeals-service-api/src/services/user.service.js

```diff
@@ -35,7 +35,10 @@ const getUsers = async (lpaCode) => {
 	}
 
 	try {
-		const cursor = await mongodb.get().collection('user').find({ lpaCode: lpaCode, enabled: true });
+		const cursor = await mongodb
+			.get()
+			.collection('user')
+			.find({ lpaCode: lpaCode, enabled: true, status: 'confirmed' });
```

Comment:
Maybe the the different statuses could go into a constants file so we could do something like status.confirmed instead. Potentially less prone typos and easier to re use.